tests/tck/features/aggregate/Agg.feature

```diff
@@ -384,6 +384,18 @@ Feature: Basic Aggregate and GroupBy
       | dst             | count |
       | "Tony Parker"   | 1     |
       | "Manu Ginobili" | 1     |
+    When executing query:
+      """
+      GO FROM "Tim Duncan" OVER like YIELD like._dst AS dst, $$.player.age AS age
+      | GROUP BY $-.dst
+      YIELD
+        $-.dst AS dst,
+        (INT)(sum($-.age)/count($-.age))+avg(distinct $-.age+1)+1 AS statistics
```

Comment:
Do we have some exception cases such as `sum(count())`?

Reply:
Yep, in Agg.feature `error check` scenario.